### DISCUSSION TITLE
LIDL - Silvercrest Lidl HG06335 motion detector can join but not reporting values like occupation or stay at null or NA

### DISCUSSION THREAD

**Discussion by adekloet** [Koenkk/zigbee2mqtt#9273] [Devices]
Hi,

Does any of you have problems with the interview process of the LIDL - Silvercrest motion detector? It joins successfully, I have even working ones that I joined in a previous older version of zigbee2mqtt but with the latest version of Zigbee2mqtt I have problem.

Tried 2 of them but the same results, does not matter if I am close or far from the coordinator. I also tried via a router in another place in the house. I tried pushing the tamper button to keep the device a live during interviewing but no success. Off course changed the battery.

My other Zigbee devices 50+ run perfectly on the Slaesh CC26522 RB coordinator. 
@Koenkk I am open to suggestions on how to join these end device motion sensors. (Did something change in the codebase for this sensor recently?)

Thanks,
Alex

MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"turn on - Lidl - Z2MQTT","ieee_address":"0xec1bbdfffee7aedd"},"type":"device_joined"}'
info MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":{"friendly_name":"turn on - Lidl - Z2MQTT"},"type":"device_connected"}'
info MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"announce","meta":{"friendly_name":"turn on - Lidl - Z2MQTT"},"type":"device_announced"}'

error Failed to configure 'Motion detect IPAD tablet turn on - Lidl - Z2MQTT', attempt 1 (Error: Bind 0xec1bbdfffee7aedd/1 genPowerCfg from '0x00124b002298904d/1' failed (AREQ - ZDO - bindRsp after 10000ms) at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35) at listOnTimeout (internal/timers.js:557:17) at processTimers (internal/timers.js:500:7))

info MQTT publish: topic 'zigbee2mqtt/Motion Sensor Lidl2 Gang Z2MQTT', payload '{"battery":100,"battery_low":false,"linkquality":147,"occupancy":false,"tamper":false,"voltage":3000}'

-----

The enities advertised stay at "null" or "NA" or the intial value and never change.

**Comment by BaldKingy** [comment]
I have the same motion detector. It is working for me out of the box until now.
When I try to decipher the error it looks for me its connected to something else?
The last info line say's it working fine.

**Comment by adekloet** [comment]
Hi @BaldKingy, thanks for your reply. I actually have 3 of them also reporting "Normal" on the current setup. The wife moved one censor (I know she shouldn't, but I need her support :)

That one sensor became in a "stalled" state. So it did not report on movement or tamper, but was still available in Zigbee2Mqtt. I tried repairing, no luck as described above. Thinking it might be broken, I took my spare one, still in the box I bought them in and tried pairing that one as a replacement. But..

Nope no luck, it pairs, but interview fails and I do see it retrying 1/2/3 times. It then either becomes staled with a random value mostly in a state of detecting and tamper on. Or it says null and NA as state except for the lqi/batt indicators (although the bat always is on 100%).

_Do you have a spare which you could give a try at reparing with the current Zigbee2Mqtt branch? (I also tried the DEV branch but no luck there also._  (my setup did not change hardware wise in the mean time, off course Core and zigbee2mqtt got it's upgrades)

I did check if no other coordinators are stealing the show, and made sure no other are in pairing state. I just now took an stock tuya wired Zigbee bridge and connected the stalled motion detector and Tuya cloud has no issue at all with the device. It functions like normal, so is not broken, but very slow and in the Tuya cloud brrrr.... 

Lost now.. any other ideas are very welcome. 

I did repair a sonoff motion detector with success. So it's definitly something in the LIDL silvercrest motion detector pairing/interview process.

**Comment by BaldKingy** [comment]
No, I only have lidl motion sensor I bought on release with some other lidl product. I like the lidl products especially the batteries are amazing long lasting.

Back to your issue. I did repair the one I have and it still functions as normal.

Two days ago I had an issue where some known devices didn't function anymore, could not repair these devices and some other strange things. A brand new lamp paired without any problem. These issues started when I replaced a battery on a Ikea remote.

There was one thing I should have done earlier, upgrade my CC2531 firmware from the default to the latest source_routing version. I was already running 21 devices, the remote was lying around with a flat battery for some time.

I presume  bringing it back to life did kill something in the coordinator default firmware. After upgrading the CC2531 (after some minutes and repairing some devices) with the latest source_routing firmware my zigbee network was working like it should :)

Can your problem be something I experienced? How many devices can your coordinator  handle?

(I'm aware with routers - bulbs etc - you can have more devices with the default firmware.)
(I'm also aware the CC2531 isn't the best coordinator. I bought it for EUR 2 or so and using it for some years without any issues I can't solve.)

**Reply by adekloet** [reply]
It's a CC2652RB from Sleash. Interesting points you make. The devices were paired with the same coordinator, but a lower version. I have reflashed the coordinator some months ago with a new version of the firmware.  I did that because I ordered some more of these ones to have a backup and and one I flashed as a router because if it's magnificent range a power to handle more than 100 devices easily.

https://slae.sh/ you'll find info. 

Back to this issue; So it might be a firmware problem in the coordinator, introduced by the update, that makes this device not want pair with al its value's present. Let me boot my backup HA and stick in a CC2531 to see what that will do wiht the current Z2M version. I can also reflash on CC2652RB to an orginal firmware and even downgrade Z2M to an older version to see if the problem goes away.

I'll report back with results. @Koenkk it looks like the last CC2652RB update fixed one issue and caused another, but we will see. I have replaced the one sensor I used for the alarm system with an IKEA one which works perfectly again.

I wish I could sniff but I gues I leak knowledge or skills to do that.

**Reply by adekloet** [reply]
Current test setup to confirm if the coordinator is the problem:

HASS OS : 6.4.
Core: 10.6
Sup: 10.0
Z2MQTT: 1.21.2-1 production

Conbee II stick: ,"majorrel":38,"minorrel":102

I can confirm that the Silvercrest Lidl HG06335 motion detector is working perfectly under the above setup.

So the problem is with the CC2652RB latest firmware upgrade: https://github.com/Koenkk/Z-Stack-firmware/commit/afd5ac4666732a7822611cbacef9ed571af99e01

@Koenkk The latest software breaks the interview proces of this particular sensor.

**Reply by adekloet** [reply]
@BaldKingy CC2531 confirmed works also, but I don't use it for production due to it's limitations.
Thanks for reacting to my post as it did help me on the right track!
